spec/policies/user_policy_spec.rb

```diff
@@ -33,6 +33,9 @@
     it { is_expected.not_to permit_action(:update) }
     it { is_expected.not_to permit_action(:edit) }
     it { is_expected.not_to permit_action(:destroy) }
+    it { expect(subject.send(:admin?)).to be false }
```

Comment:
This is a protected method.  When I do `it { expect(subject.admin?).to be false }` I get the below message.

```
Failures:

  1) UserPolicy when user has the moderator role
     Failure/Error: it { expect(subject.admin?).to be false }

     NoMethodError:
       protected method `admin?' called for #<UserPolicy:0x007fe8c111c180>
     # ./spec/policies/user_policy_spec.rb:36:in `block (3 levels) in <top (required)>'
```